actions/oryx-dockerfile/README.md

```diff
@@ -0,0 +1,207 @@
+# GitHub Action for generating a Dockerfile to build and run Azure Web Apps
+
+With the Azure App Service Actions for GitHub, you can automate your workflow to deploy [Azure Web Apps](https://azure.microsoft.com/en-us/services/app-service/web/) using GitHub Actions.
+
+Get started today with a [free Azure account](https://azure.com/free/open-source)!
+
+This repository contains the GitHub Action for [generating a Dockerfile to build and run Azure Web Apps](./action.yml) using the [Oryx](https://github.com/microsoft/Oryx) build system. Currently, the following platforms can be built using this GitHub Action:
+
+- .NET Core
+- Node
+- PHP
+- Python
+
+The generated Dockerfile follows a template similar to the following:
+
+```
+ARG RUNTIME=<PLATFORM_NAME>:<PLATFORM_VERSION>
+
+FROM mcr.microsoft.com/oryx/build:<BUILD_TAG> as build
+WORKDIR /app
+COPY . .
+RUN oryx build /app
+
+FROM mcr.microsoft.com/oryx/${RUNTIME}
+COPY --from=build /app /app
+RUN cd /app && oryx
+ENTRYPOINT ["/app/run.sh"]
+```
+
+Once this Dockerfile is produced, it can be built and pushed to a registry, such as [Azure Container Registry](https://azure.microsoft.com/en-us/services/container-registry/), and ran at a later time to deploy the Azure Web App.
```

Comment:
instead of "and ran at a later time", how about "and _used_ at a later time" since we're talking about deployment ?